kr8s/_api.py

```diff
@@ -111,6 +111,7 @@ async def _create_session(self) -> None:
             headers=headers,
             verify=await self.auth.ssl_context(),
             timeout=self._timeout,
+            follow_redirects=True,
```

Comment:
If a baseurl is set we may get redirected. By default `httpx` doesn't follow redirects, but we probably always want to.